### DISCUSSION TITLE
播放视频时一抖一抖的

### DISCUSSION THREAD

**Discussion by likepoems** [AlistGo/alist#1621] [General]
使用阿里网盘和onedrive挂载时，播放视频画面抖动，我在阿里云盘上直接播放是正常的，视频文件没问题。
具现链接：https://pan.likepoems.com/local/videos/你的名字.mp4

**Comment by cobojr** [comment]
需要修改nginx里的一个配置文件，因为做了反代加上ssl的关系，你可以关掉ssl试试，或者已端口来访问试试看

**Reply by likepoems** [reply]
试过，还是不行。

**Comment by xuegodot** [comment]
个人经验。
这个好像是网盘问题。比如夸克。如果文件很大几个G的时候。就是一抖一抖的。我都是下载回来看。pikpak也是这样。不知道是不是我网速不够。

**Reply by banshicy** [reply]
跟网盘没关系，自己下个播放器，在播放器里面放就没事了，视频和浏览器的原因。

**Comment by banshicy** [comment]
跟网盘没关系，自己下个播放器，在播放器里面放就没事了，视频和浏览器的原因。

**Reply by basten1899** [reply]
网页播放器的问题吧  不支持插帧?
